docs/visual-basic/programming-guide/concepts/linq/maintaining-name-value-pairs.md

```diff
@@ -94,22 +94,20 @@ Console.WriteLine(root)
   <Bottom>122</Bottom>  
   <Right>300</Right>  
   <DefaultColor>Color.Red</DefaultColor>  
-</Root>  
-----  
+</Root>
```

Comment:
The dashed lines need to stay - they are the output from lines 76 and 81. Or we should remove lines 76 and 81 instead.

Reply:
But you could change the snippet language to "txt" instead of "xml".